docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -30,40 +36,121 @@ is a small, initial footprint that has a publicly facing network and accessible
+workstations, we cloned the
+[Doubtfire-deploy-GCP repository](https://github.com/thoth-tech/doubtfire-deploy-GCP) and modified
+the necessary files. We then used docker compose and Docker to run and deploy containers.
+
+![doubtfire-localhost-compose](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost-compose.png "docker compose output")
+
+Success! we have Docker containers running locally.
+
+![doubtfire-localhost-docker](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost-docker.png "Docker containers running")
+
+Success again! We have OnTrack being hosted locally and is accessible via https://localhost:443 .
+
+![doubtfire-localhost](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost.png "Doubtfire running on localhost")
+
+### Google Compute Engine instance
+
+Once we determined the configuration changes required to be able to run locally (localhost), we then
+needed to determine how to create and deploy a Compute Engine server instance in GCP that we could
+use to deploy Doubtfire.
+
+We started with a small, initial footprint and deployed a basic virtual machine (VM) instance with
+following details -
+
+- Name: instance-1
+- Zone: australia-southeast2-a
+- Machine type: e2-micro
+- CPU platform: Intel Broadwell
+- OS Image: Centos 7 (Image name centos-7-v20220406)
+- Boot disk: 20GB SCSI (Balanced persistent disk, Google-managed encryption)
+- Network interfaces: nic0 (with internal & external IP addresses)
+- Firewalls: HTTP, HTTPS enabled
+- GPUs/Display device: None, disabled
+
+You can find the equivalent REST for our original instance from Google Compute Engine
+[here](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-rest.json).
+
+Once we had the instance up and running, we connected to the instance using command-line shell via
+SSH. In the Google Cloud console, you can view the options to connect by clicking the drop-down menu
+beside _Connect SSH_ on the instance view.
+
+![gcp-instance-connect](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-connect.png "Instance connection options")
+
+Alternatively, you can SSH to the instance via gcloud CLI -
+
+```
+gcloud compute ssh --zone "australia-southeast2-a" "instance-1"  --project "sit-22t1-ontrack-deplo-d026375"
+```
+
+There were a few packages that needed to be installed on the host, such as -
+
+- yum-utils
+- yum-config-manager (required to setup docker-ce repo, see
+  [Docker docs](https://docs.docker.com/engine/install/centos/) for Centos)
+- docker-ce, docker-ce-cli, containerd.io, docker-complose-plugin
+- git
+- openssl
+- nano (optional)
+
+Once installed using _yum_, we had the minimum requirements to get started on the Doubtfire
```

Comment:
**suggestion:** We [reserve](https://github.com/thoth-tech/handbook/blob/main/docs/processes/documentation/writing-style-guide.md#text-formatting) italic formatting for titles and legislations, or user input.

```suggestion
Once installed using `yum`, we had the minimum requirements to get started on the Doubtfire
```